lib/modules/apostrophe-pieces/lib/routes.js

```diff
@@ -34,6 +35,27 @@ module.exports = function(self, options) {
     return self.retrieveResponse(req, res, null, req.piece);
   };
 
+  self.routes.suggestions = async function(req, res) {
+    try {
+      const {term, filters} = req.body;
+
+      if (!term || typeof filters !== 'object') {
+        const errorMsg = !term
+          ? 'A search term must be provided.'
+          : 'Filters must be an object containing search filters.';
+
+        return self.apiResponse(res, errorMsg);
```

Comment:
This is fine. You could have used `self.apiRoute` to remove a little more code I think.

Reply:
I didn't use it initially because it wasn't used in this file.
Just modified it.